### ISSUE TITLE
setup.sh bugs

### ISSUE THREAD

**Issue by soundray** [MIRTK/DrawEM#34]
It seems that the two lines towards the end of setup.sh

```
echo "export DRAWEMDIR=$DRAWEMDIR" >> $code_dir/parameters/path.sh
echo "export PATH=${MIRTK_build}/bin:"'${PATH}' >> $code_dir/parameters/path.sh
```

attempt to write to a path that does not exist. Should the variables be `$DRAWEMDIR` instead of `$code_dir`?

**Comment by Emilyjane994** [comment]
Hi, were you able to install DrawEM? i am having difficulty with the setup.sh

**Comment by soundray** [comment]
Yes, I installed commit `d2ff4e3` recently. In `setup.sh`, I adapted `compiler_version_required` and set `MIRTK_version` to `973ce2fe3f9508dec68892dbf97cca39067aa3d6`. I also modified the line `set_if_undef MIRTK_cmake_flags=`; this depends strongly on local context.

Please describe your difficulties in detail -- I'm not a developer, but I have some experience muddling through the installation.

**Comment by Emilyjane994** [comment]
I keep running into an issue when running make for MIRTK. The OneTBB-2020-U3 was depreciated so I installed OneTBB 2021.5.0.  The issue is that MIRTK expects tbb::task, which seems to have been removed in TBB 2021. 

I amended the script for the compiler_version_required and updated the MIRTK-version as per your last comment, but I am still running into this issue with TBB.
